lib/salus/scanners/yarn_audit.rb

```diff
@@ -7,12 +7,78 @@
 
 module Salus::Scanners
   class YarnAudit < NodeAudit
-    AUDIT_COMMAND = 'yarn audit --json'.freeze
```

Comment:
Can we leave a comment here, maybe add a link to https://github.com/yarnpkg/yarn/issues/7404 so we remember why we changed this?